charts/cni-metrics-helper/values.yaml

```diff
@@ -28,3 +28,9 @@ serviceAccount:
     # eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME  
 
 resources: {}
+
+revisionHistoryLimit: 10
+
+podSecurityContext: {}
+
+securityContext: {}
```

Comment:
nit: maybe we should call this `containerSecurityContext` to differentiate it?

Reply:
Some charts use `securityContext` & `containerSecurityContext`, others use `podSecurityContext` and `securityContext` so just a case of settling on a convention. I can update the name.